src/components/styles/PageSection.styled.ts

```diff
@@ -48,6 +67,7 @@ export const Title = styled.h2<{ events?: boolean; fontType: FontType }>`
 export const Body = styled.p<{ events?: boolean; fontType: FontType }>`
   ${(props) => `color: ${props.theme.palette.common.gray};`}
   ${fontTypeCss}
+  font-size: 1.1rem;
```

Comment:
if font size is written here, is there a need to use the fontTypeCss?

Reply:
yes there is a difference in text with it removed, fontTypeCss controls its fontweight too